packages/webhook-utils/src/disable-webhook.ts

```diff
@@ -0,0 +1,17 @@
+import { Client } from "urql";
+import { modifyAppWebhook } from "./operations/modify-app-webhook";
+
+interface disableWebhookArgs {
+  client: Client;
+  webhookId: string;
+}
+
+export const disableWebhook = async ({ client, webhookId }: disableWebhookArgs) => {
+  const response = await modifyAppWebhook({
```

Comment:
probably error should be handled?

Reply:
Throwing an error is handled by modifyAppWebhook